src/ui/public/management/__tests__/section.js

```diff
@@ -94,6 +94,15 @@ describe('ManagementSection', () => {
     it('returns undefined if un-registered', () => {
       expect(section.getSection('unknown')).to.be(undefined);
     });
+
+    it('returns sub-sections specified via a /-separated path', () => {
+      section.getSection('about').register('time');
+      expect(section.getSection('about/time')).to.be.a(ManagementSection);
```

Comment:
Adding a check for the "old way" of getting this would probably be good:

```js
expect(section.getSection('about').getSection('time')).to.equal(section.getSection('about/time'))
```